docs/docsite/rst/dev_guide/developing_python3.rst

```diff
@@ -274,67 +374,42 @@ coded to expect bytes on Python-2 and text on Python-3.
 Tips, tricks, and idioms to adopt
 =================================
 
-Exceptions
-----------
+Old Exception Syntax
```

Comment:
"old" is a relative term, I think a more precise title using versions would be better.